### PR TITLE
[Fix] Toaster position #291

### LINKED ISSUE #291: Update notification position and timing
Should be positioned at top right of "app" frame i.e. to the left of an open transactions drawer.

Should only display for 3 seconds.

### PR DESCRIPTION
**Pull request by codyenokida**
Closes #291 

- fixed toaster position

### CODE REVIEW

File: src/components/Toasts/Toaster.tsx [hunk 1]
```diff
@@ -13,12 +13,19 @@ const ToasterWrapper = styled.div`
...
+const Toaster = ({ open }: ToasterPropType) => (
   <ToasterWrapper>
     <T
       position="top-right"
       containerStyle={{
-        right: "80px",
+        right: open ? "25rem" : "1rem",
+        transform: `${(open: boolean) =>
+          open ? "translateX(0)" : "translateX(24rem)"}`,
+        transition: "transform 0.3s ease-in-out;",
```
Comment (robbestad) [inline src/components/Toasts/Toaster.tsx:28]: that semicolon at the end shouldn't be there

**Review by robbestad — CHANGES_REQUESTED** [review]
Looks great, only a very minor thing

**Review by robbestad — APPROVED** [review]

**Review by piersss — APPROVED** [review]
This works

**Comment by gpxl-dev** [comment]
Think we should merge #317 first or we'll have two lots of conflicts to fix.

**Review by gpxl-dev — APPROVED** [review]

**Pull request merged**